src/main/resources/db/changesets/20181203_studytable.yaml

```diff
@@ -0,0 +1,109 @@
+databaseChangeLog:
+  - changeSet:
+      id: study_init
+      author: jhert
+      changes:
+        - createTable:
+            tableName: study
+            columns:
+              - column:
+                  name: id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(1000)
+                  constraints:
+                    nullable: false
+
+        - createTable:
+            tableName: table_def
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(128)
+                  constraints:
+                        nullable: false
+
+        - createTable:
+            tableName: column_def
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: table_id
+                  type: int
+              - column:
+                  name: name
+                  type: varchar(256)
+              - column:
+                  name: type
+                  type: varchar(128)
```

Comment:
We know we will innovate around types, moving to support semantic types. We can either accept that we will have to evolve the schema when we get there, or put a "type namespace" attribute here. For now, it would have one value (primitive), but as we extend the typing system, we can add different namespaces from which we understand types.

Or just add a TODO...

Reply:
I'm making a JIRA story around this -- I'd prefer to evolve it when we get there. TODOs aren't as useful in migrations since we most likely won't be revisiting this file.

Reply:
👍